articles/multifactor-authentication/google-auth/dev-guide.md

```diff
@@ -0,0 +1,97 @@
+# Google Authenticator for Developers
```

Comment:
This info is also repeated, I guess it might make sense, not sure. See my comment for DUO